### PR TITLE
Addition of a design doc for `find_green_build` functionality.

### PR DESCRIPTION
**Pull request by david-mcmahon**
This comes after its implementation but want to include some detail on the design in a doc here.
Lets keep the discussion for this limited to existing functionality.  Please ask for clarification on anything and open new issues as needed.

cc @kubernetes/test-infra-maintainers

### CODE REVIEW

**Comment by philips** [comment]
cc @ethernetadan

File: contributors/design-proposals/release-test-signal.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+- ci-kubernetes-e2e-gke-serial      #233    #2319   PASSED
+- ci-kubernetes-e2e-gke             #1834   #2330   PASSED
+- ci-kubernetes-e2e-gke-slow        #1041   #2330   PASSED
+
+JENKINS_BUILD_VERSION=v1.6.0-alpha.0.2330+75dfb21018a7c3
+RELEASE_VERSION[alpha]=v1.6.0-alpha.1
+RELEASE_VERSION_PRIME=v1.6.0-alpha.1
+```
+
+### V1
```
Comment (ixdy) [inline contributors/design-proposals/release-test-signal.md]: nit: capitalization of "v" is inconsistent between v1 and v2.

Comment (david-mcmahon) [inline contributors/design-proposals/release-test-signal.md]: PTAL

**Pull request merged**